MIGRATING.md

```diff
@@ -1,96 +0,0 @@
-# Migrating from Version 1 to Version 2
```

Comment:
Is this file linked to from anywhere? I think it may be (from a blog post or dev guides?).  I'd suggest keeping it around but adding some messaging around the maintaince mode schedule for version 2 + info on where to find guides for migrating to V3 or do what you did with the UPGRADING.md and move this to version-2 only with a link.

Reply:
the exact same file is in version-2 and yes it's linked in multiple places: https://github.com/aws/aws-sdk-ruby/blob/version-2/MIGRATING.md